secrets/values.yaml

```diff
@@ -0,0 +1,26 @@
+  mongodbReplicaSetKey: AgBypFPFGNgvHBcKTMoNNOHY3oZFVPvo+Nc/datBWWuCujHncaSQHXTbABuXZjW1XN+9LDGL5iSQr0S2m0jAp+d0GX2e+Zyw6aRTG8kZqwhy+8SjdPhvOrLH8CCWdk02dCUJhekjNM4IL9QgV3o7k4Hy2vrLVMkwkaNzhJjcon7LhAbRbEZbMXUvk/G4tac09Wz/m/pqtpzs+TZoSfjsvQQQXDZCqqavxir3wNPogEV98B5cHW1adx6zHaXya5hldW8bX5UxOmSUA76Y+20HQ6cfyGpZbIVgtuGk0TAkytOR0MJ2plrr7jqhGkeYUqhBAcp7w7kPmGq0JQyj7e0xeGHQuixUsfut+V0bfjartccCTtmwLnIb38WndbG1aavUwhefGmaPBFn4JzD2kohpZSOScRYF2N9G98jzcfhS+qVX5c5fn08ULZI2bBXZ4PBuZhtdy4lUMOuG4bY2O/7KxOcQ9TN2EHa1EsVukkHiUMu4z5X5rsZfkXXdMYYoTXqJmO0Zevu0jGMbuQWmR3qs07uGVxT1x3UvOCQ/g8rCW8toQX30uCW/az3bTFb4toRU/EYqCx2+MaGoDucQTDU9krfZlKZ4NFAo6+t9vOInNsBJxXPcDbtRr5phMVbINVt/ZuX1Nl4JoXBoF/jV0MIgvlGYPp9/NIxKs7IhuAxPKgyyQT9rzZcWoY/1tr/jpMltHgpySszxMLI2IfmF
+
+grafana:
+  grafanaUsername: AgCUAkfP0YRSeM/OR43BkElJPj2L+JjVjaD20rITR610uulm5d+7wkoy7ds7uYARSrw5l6Pd99hYtQZof+Oj262TUPRu8hc7tkZnkHv6NS57l9XyGUkEYaat7m9Wm6WmUqSpV/5+yvhUfm3Jt7mxl73btVIK1vTo2ntj/MBvgKHXGwaczE13JS0ecl1+DK927sxru2nIedzcANmKimq5zJfudWdIE/mi8hs1/KlAVGz5w3jkBDSYAPlxNtX9ZeKt8cvWVKousuh+1QWLWRTJuPaK71nCnqY18lzUWOUTQKf/m1/3XbL4YLrE8uTf2c3AGRkGz5PWB5GA/34Mxtt38FU8tDWXWNhK01j6KtOv9s+Tgo/s4vV2Y+UBFxmVX3atd7arCCbxfakQUgSGA7nvpq3jvmUiz9PywgawTUkSC+0EDFkc8o1p80O82vrjBOugGNHYLtkF6qiDI7cPUs212heS+T5mDIJm5FqYUaKlRIqRx6K0ZHpK/W7rV6P5epgu6qSu3E6AqDEZpasjnETdrGH5dgd3HyILb3MJ3iLub7sS5jtGwPrqz31iWixAs22WZakF1p9gMaxuOFoMGoELAQYpZbxnECu5DymW+3AmunU9hpLmdRTgLcKFhi/4i45qg/0TsTf0CQIwMprNABI0OLGHWb+xEdVJEpYcDjIebv5SQpBWOUpqmdpAMtTnbqsDifO8PJhfxnH5+0hnQQ==
+  grafanaPassword: AgCUAkfP0YRSeM/OR43BkElJPj2L+JjVjaD20rITR610uulm5d+7wkoy7ds7uYARSrw5l6Pd99hYtQZof+Oj262TUPRu8hc7tkZnkHv6NS57l9XyGUkEYaat7m9Wm6WmUqSpV/5+yvhUfm3Jt7mxl73btVIK1vTo2ntj/MBvgKHXGwaczE13JS0ecl1+DK927sxru2nIedzcANmKimq5zJfudWdIE/mi8hs1/KlAVGz5w3jkBDSYAPlxNtX9ZeKt8cvWVKousuh+1QWLWRTJuPaK71nCnqY18lzUWOUTQKf/m1/3XbL4YLrE8uTf2c3AGRkGz5PWB5GA/34Mxtt38FU8tDWXWNhK01j6KtOv9s+Tgo/s4vV2Y+UBFxmVX3atd7arCCbxfakQUgSGA7nvpq3jvmUiz9PywgawTUkSC+0EDFkc8o1p80O82vrjBOugGNHYLtkF6qiDI7cPUs212heS+T5mDIJm5FqYUaKlRIqRx6K0ZHpK/W7rV6P5epgu6qSu3E6AqDEZpasjnETdrGH5dgd3HyILb3MJ3iLub7sS5jtGwPrqz31iWixAs22WZakF1p9gMaxuOFoMGoELAQYpZbxnECu5DymW+3AmunU9hpLmdRTgLcKFhi/4i45qg/0TsTf0CQIwMprNABI0OLGHWb+xEdVJEpYcDjIebv5SQpBWOUpqmdpAMtTnbqsDifO8PJhfxnH5+0hnQQ==
+
+  datasources:
+    timescale:
+      credentials:
+        password: *timescale-password
+
+  datasourcesNotEncrypted:
```

Comment:
Cant we do sth about this? it is plain text here...

Reply:
Where is the plain text? That's a link to the name of the password